### PR TITLE
FE-59 Fix: translated zuri chat choose-workspace page

### PR DESCRIPTION
**Pull request by Bobbyjsx**
# Fixes Issue/Linear Ticket

( This could be an existing issue or a linear ticket )

**My PR Closes #issue_number_here** or
**My PR Fixes ID-linear_ticket_number_here**

# Changes proposed

## What were you told to do?

** I was told to make my right to left languages align properly**

## What did you do?

**I added a text-align variable to all pages I translated. and styled them accordingly  **
# Check List (Check all the applicable boxes)

🚨Please review the [style guide for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/CONTRIBUTING.md) to this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/ Videos
Step0 jsx - zc_main-bobb yy - Visual Studio Code 11_28_2022 12_31_29 AM

### CODE REVIEW

**Comment by JulianaSau** [comment]
This code is repeated several times. It's not a fix if it needs to be added manually to all pages. Please come up with a better solution using the DRY principle as a guide

**Comment by akarasauce** [comment]
Necessary changes have been made as requested.

File: packages/main/src/pages/protected/choose-workspace/ChooseWorkspace.jsx [hunk 1]
```diff
@@ -35,9 +35,20 @@ export default function Index() {
     }
   }, []);
   const history = useHistory();
+  //text align right for arabic and hebrew
```
Comment (ickynavigator) [inline packages/main/src/pages/protected/choose-workspace/ChooseWorkspace.jsx:38]: i'm pretty sure this function is already implemented

File: packages/main/src/pages/protected/create-workspace/CreateWorkspace.jsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ export default function Index() {
   });
 
   return (
-    <>
+    <div>
```
Comment (ickynavigator) [inline packages/main/src/pages/protected/create-workspace/CreateWorkspace.jsx:19]: why'd you nest it in a div?
aren't you cluttering?

File: packages/main/src/pages/protected/create-workspace/Step0.jsx [hunk 3]
```diff
@@ -33,7 +33,7 @@ export default function Index() {
   }, []);
 
   return (
-    <>
+    <div>
```
Comment (ickynavigator) [inline packages/main/src/pages/protected/create-workspace/Step0.jsx:36]: same here?

File: packages/main/src/translations/translation-dictionary/pages/Workspace.js [hunk 4]
```diff
@@ -238,7 +238,7 @@ export default [
     pt: "Isto pode ser qualquer coisa - um projecto, campanha, evento, ou o negócio que está a tentar fechar"
   },
   {
-    key: "create_workspace_eighteen",
+    key: "create_workspace_eighteen\n",
```
Comment (ickynavigator) [inline packages/main/src/translations/translation-dictionary/pages/Workspace.js:241]: why would you add that to the key?

File: packages/main/src/pages/protected/choose-workspace/ChooseWorkspace.jsx (hunk 1 above)
Comment (akarasauce) [reply packages/main/src/pages/protected/choose-workspace/ChooseWorkspace.jsx:38]: yes i've removed it

File: packages/main/src/pages/protected/create-workspace/Step0.jsx (hunk 3 above)
Comment (akarasauce) [reply packages/main/src/pages/protected/create-workspace/Step0.jsx]: it has been resolved

File: packages/main/src/pages/protected/choose-workspace/ChooseWorkspace.jsx (hunk 1 above)
Comment (JulianaSau) [reply packages/main/src/pages/protected/choose-workspace/ChooseWorkspace.jsx:38]: You've not removed this textAlign function @akarasauce

**Review by JulianaSau — CHANGES_REQUESTED** [review]

**Review by akarasauce — COMMENTED** [review]
Resolved

File: packages/main/src/pages/protected/choose-workspace/ChooseWorkspace.jsx (hunk 1 above)
Comment (Bobbyjsx) [reply packages/main/src/pages/protected/choose-workspace/ChooseWorkspace.jsx]: please re-review my pull request 🥺@JulianaSau

**Review by JulianaSau — APPROVED** [review]

**Pull request merged**